services/buildkite/buildkite.tester.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const ServiceTester = require('../service-tester');
+const t = new ServiceTester({ id: 'buildkite', title: 'Buildkite Builds' });
+const { invalidJSON } = require('../response-fixtures');
+module.exports = t;
+
+t.create('buildkite invalid pipeline')
+  .get('/unknown-identifier/unknown-branch.json')
+  .expectJSON({ name: 'build', value: 'not found' });
+
+t.create('buildkite valid pipeline')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489/master.json')
+  .expectJSON({ name: 'build', value: 'passing' });
+
+t.create('buildkite valid pipeline skipping branch')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489.json')
+  .expectJSON({ name: 'build', value: 'passing' });
```

Comment:
Sorry just noticed this also,
Could you update the relevant test to use:
```javascript
  .expectJSONTypes(Joi.object().keys({
    name: 'build',
    value: Joi.equal('failing', 'passing', 'unknown')
  }));
```
This just ensures our test won't fail if the service that we are testing against fails :smile: